htsohm/runDB_declarative.py

```diff
@@ -47,7 +48,7 @@ def __init__(self, run_id, generation, dummy_test_result):
         self.generation = generation
         self.dummy_test_result = dummy_test_result
 
-with open('database.yaml', 'r') as yaml_file:
+with open('database.sample.yaml', 'r') as yaml_file:
```

Comment:
Nope. That's totally not the point of it being a sample! :)

Copy and make changes to database.yaml locally.